### PR TITLE
Add warning about usage of PreSendRequestHeaders and PreSendRequestContent

### PR DESCRIPTION
**Pull request by yaananth**
see https://github.com/aspnet/Docs/pull/5054/files

I don't know where we place warnings generally, but I placed it under "Remarks"

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
Looks good.

File: xml/System.Web/HttpApplication.xml [hunk 1]
```diff
@@ -2048,7 +2048,8 @@ This method provides extensibility to the ASP.NET pipeline to make it easy for d
  The <xref:System.Web.HttpApplication.PreSendRequestContent> event may occur multiple times.  
   
  For more information about how to handle events, see [NIB: Consuming Events](http://msdn.microsoft.com/en-us/01e4f1bc-e55e-413f-98c7-6588493e5f67).  
-  
+ > [!WARNING]
+ > Do not use `PreSendRequestContent` with managed modules that implement `IHttpModule`. Setting these properties can cause issues with asynchronous requests. We have seen instances where the combination of Application Requested Routing(ARR) and websockets lead to access violation exceptions (iiscore!W3_CONTEXT_BASE::GetIsLastNotification+68 in iiscore.dll has caused an access violation exception (0xC0000005)) that lead to w3wp crashing.
```
Comment (mairaw) [inline xml/System.Web/HttpApplication.xml]: missing space between term and acronym:  Application Requested Routing (ARR)

The use of we is not recommended unless we're using it to refer to we Microsoft recommends. Also the last sentence is a bit hard to parse giving it has lead 2x and that info on parentheses breaking the flow. I'm wondering if that info is really needed.

Maybe something like this?
"The combination of Application Requested Routing (ARR) and websockets might lead to access violation exceptions that can cause w3wp to crash. For example, iiscore!W3_CONTEXT_BASE::GetIsLastNotification+68 in iiscore.dll has caused an access violation exception (0xC0000005)."

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks for your contribution @yaananth. I've left some feedback for you to consider. The same comment would apply to the 2nd change.

File: xml/System.Web/HttpApplication.xml (hunk 1 above)
Comment (Rick-Anderson) [reply xml/System.Web/HttpApplication.xml]: @mairaw  I think that's the MS recommendation.

I like your rewrite.

**Comment by yaananth** [comment]
@mairaw Addressed.
Also https://github.com/aspnet/Docs/pull/5067 for the other repo

**Review by mairaw — APPROVED** [review]
Thanks for making the suggested changes @yaananth. I'll merge as soon as I can verify the build and changes should be live in the next day or so.

File: xml/System.Web/HttpApplication.xml (hunk 1 above)
Comment (mairaw) [reply xml/System.Web/HttpApplication.xml]: Yes, it's in the style guide @Rick-Anderson. I'm glad you liked it. Thanks!
